syntax = 'proto3';

package ftfbroker.protobuf.ftf;
import "ftfbroker/protobuf/common/metaData.proto";

message AutomateStock {
    oneof meta {
        ftfbroker.protobuf.common.MetaDataV1 meta_v1 = 1;
    }
    oneof payload {
        AutomateStockV1 payload_v1 = 2;
    }
}

message AutomateStockV1 {
    float distance_cm = 1;
    int64 num_bottles = 2;
}
